### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/arrayMethodJoin.js [hunk 1]
```diff
@@ -5,7 +5,32 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let result = '';
+    let sep = separator;
+
+    if (sep === undefined) {
+      sep = ',';
+    }
```
Comment (Yeliseiev) [inline src/arrayMethodJoin.js:13]: I think you forgot that you can assign a dafault value. Example: function(separator = ',')

**Review by Yeliseiev — APPROVED** [review]
GJ. Too much 'if' as for me, but still good work.

**Review by silchyn — CHANGES_REQUESTED** [review]
Good point above

File: src/arrayMethodJoin.js [hunk 2]
```diff
@@ -4,8 +4,29 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let result = '';
+    const sep = separator;
```
Comment (silchyn) [inline src/arrayMethodJoin.js:9]: Do you need sep?

**Review by silchyn — CHANGES_REQUESTED** [review]

**Review by silchyn — APPROVED** [review]
Cool!

**Pull request closed without merging**